mne/viz/tests/test_topo.py

```diff
@@ -108,6 +109,14 @@ def return_inds(d):  # to test function kwarg to zorder arg of evoked.plot
                       topomap_args=dict(proj='reconstruct'))
     plt.close('all')
 
+    # test sEEG (gh:8733)
+    evoked.del_proj().pick_types('mag')  # avoid overlapping positions error
+    mapping = {ch_name: 'seeg' for ch_name in evoked.ch_names}
+    with warnings.catch_warnings():  # unit has changed from T/m to V
+        warnings.simplefilter('ignore')
+        evoked.set_channel_types(mapping)
```

Comment:
Typically with these we just do:
```suggestion
    with pytest.warns(RuntimeWarning, match='unit changed')
        evoked.set_channel_types(mapping)
```
Does the same (plus a bit more, checking that we got the expected one) with fewer lines/imports